test/browserify-test.js

```diff
@@ -0,0 +1,29 @@
+/* Copyright (c) 2012-2016 LevelUP contributors
+ * See list at <https://github.com/level/levelup#contributing>
+ * MIT License <https://github.com/level/levelup/blob/master/LICENSE.md>
+ */
+
+var common  = require('./common')
+
+  , assert  = require('referee').assert
+  , refute  = require('referee').refute
+  , buster  = require('bustermove')
+  , browserify = require('browserify')
+  , path = require('path')
+
+var PACKAGE_JSON = path.resolve(path.join(__dirname, '..', 'package.json'))
+
+buster.testCase('Browserify Bundle', {
+  'does not contain package.json': function (done) {
+    var b = browserify('.', {browserField: true})
```

Comment:
i'd prefer `browserify(path.join(__dirname, '..'), {browserField: true})` so this test works independent of your current working directory